sample-app/redisearch-node-rest/server.js

```diff
@@ -1,22 +1,19 @@
-const express = require('express');
-const cors = require('cors');
-const app = express();
-const bodyParser = require('body-parser');
-const serverPort = process.env.SERVER_PORT || 8086;
-
+import express from 'express';
+import cors from 'cors';
+import SearchService from './NodeSearchService.js';
 
+const app = express();
+app.use(cors());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
 
-const SearchService = require('./NodeSearchService');
+const serverPort = process.env.SERVER_PORT || 8086;
 const searchService = new SearchService();
 
-app.use(bodyParser.urlencoded({ extended: true }));
-app.use(cors());
-app.use(bodyParser.json());
-
-app.get('/api/1.0/movies/search', (req, res) => {
+app.get('/api/1.0/movies/search', async (req, res) => {
   const queryString = req.query.q;
-  const offset = Number((req.query.offset)?req.query.offset:'0');
-  const limit = Number((req.query.limit)?req.query.limit:'10');
+  const offset = Number((req.query.offset) ? req.query.offset : 0);
+  const limit = Number((req.query.limit) ? req.query.limit : 10);
```

Comment:
change to `req.query.limit ?? 10`

edit: and validate it's not over 500 (or any other number)